src/renderStatic.js

```diff
@@ -44,11 +44,8 @@ const renderStatic = (locals, callback) => {
     const assets = Object.keys(webpackAssets);
     const cssAssets = assets.filter(p => /\.css$/.test(p)).map(p => locals.publicPath + p);
     const jsAssets = assets.filter(p => /\.js$/.test(p)).map(p => locals.publicPath + p);
-    const faviconstats = webpackAssets[locals.faviconstatsFilename];
```

Comment:
Er ikke sikker på om denne skal bort. Jeg ser at favicons blir laget med den nye webapp-webpack-plugin, men `yarn serve` klager på at den ikke finner favicons... Vanligvis kan den jo legge inn `<link ...` via HtmlWebpackPlugin (og det funker nok med `yarn start`), men vi bruker jo bare static site generator for `yarn build`, så da må det legges inn manuelt. Nå vet jeg ikke om den "gamle" måten her funker, det må jo sjekkes, men det bør jo legges inn på ett eller annet vis...

Reply:
Se https://github.com/Skagevang/codeclub-viewer/pull/17